packages/application-generic/src/services/cache/cache.service.ts

```diff
@@ -157,13 +158,7 @@ export class CacheService implements ICacheService {
   private getTtlInSeconds(options?: CachingConfig): number {
     const seconds = options?.ttl || this.cacheTtl;
 
-    return this.ttlVariant(seconds);
-  }
-
-  private ttlVariant(num): number {
-    const variant = this.TTL_VARIANT_PERCENTAGE * num * Math.random();
-
-    return Math.floor(num - (this.TTL_VARIANT_PERCENTAGE * num) / 2 + variant);
+    return addJitter(seconds, this.TTL_VARIANT_PERCENTAGE);
```

Comment:
Is adding Jitter the term of what we want to add here? 
I am asking because it is a new term for me 😅

Reply:
Its a common term over the statistical one, variance. 
When I abstracted this jitter was the better choice.

Reply:
AWS uses `jitter` to define the variable time to wait between retries, so it is rightly used.